pkg/manifest/os_test.go

```diff
@@ -248,3 +248,21 @@ func TestFirstBootDoesNotIncludeMachineIdStage(t *testing.T) {
 	st := findStage("org.osbuild.machine-id", pipeline.Stages)
 	require.Nil(t, st)
 }
+
+func TestLanguageIncludesLocaleStage(t *testing.T) {
+	os := NewTestOS()
+
+	os.Language = "en_US.UTF-8"
+
+	pipeline := os.serialize()
+	st := findStage("org.osbuild.locale", pipeline.Stages)
+	require.NotNil(t, st)
+}
+
+func TestLanguageDoesNotIncludeLocaleStage(t *testing.T) {
+	os := NewTestOS()
+
+	pipeline := os.serialize()
+	st := findStage("org.osbuild.locale", pipeline.Stages)
+	require.Nil(t, st)
+}
```

Comment:
This commit can be squashed with the first one.  They even have almost the same message already.

Reply:
Done.